src/loaders/gtdb/gtdb_genome_attribs_loader.py

```diff
@@ -18,62 +15,42 @@
 
 Prepare GTDB genome attributes data in JSON format for arango import.
 
-This script will first parse genome features from the GTDB metadata files. Those features should be put into
-EXIST_FEATURES. 
+This script parses genome features from the GTDB metadata files. Those features should be put into
+loader_common_names.SELECTED_FEATURES.
 
-It will then try to compute other(non-existing) features. The name of non-existing features should be put into 
-NON_EXIST_FEATURES. Each non-existing features should have a corresponding function defined in gtdb_genome_attribs_helper.
-NOTE: the function name should be exactly the same as the non-existing feature name.
+NOTE: The Document Key ("_key") in Arango DB is generated by applying a hash function to a string consisting of the 
+kbase_collection, load_version, and genome id parsed from the accession field from the GTDB metadata file. 
+(e.g. GTDB_r207.kbase.1_GCA_000016605.1, GTDB_r207.kbase.1_GCA_000169995.1)
 
-Any features that are used for computing the non-existing features should be added to HELPER_FEATURES. They will be
-eventually dropped from the result dataframe.
+usage: gtdb_genome_attribs_loader.py [-h] --load_ver LOAD_VER [--kbase_collection KBASE_COLLECTION] [-o OUTPUT]
+                                     load_files [load_files ...]
 
+PROTOTYPE - Prepare GTDB genome statistics data in JSON format for arango import.
 
-usage: gtdb_genome_attribs_loader.py [-h]
-                                   --load_version
-                                   LOAD_VERSION
-                                   [--kbase_collection KBASE_COLLECTION]
-                                   [-o OUTPUT]
-                                   load_files
-                                   [load_files ...]
+options:
+  -h, --help            show this help message and exit
 
+required named arguments:
+  load_files            GTDB genome metadata files
+  --load_ver LOAD_VER   KBase load version. (e.g. r207.kbase.1)
 
-e.g. gtdb_genome_attribs_loader.py bac120_metadata_r207.tsv ar53_metadata_r207.tsv --load_version 207
-     gtdb_genome_attribs_loader.py bac120_metadata_r207.tsv ar53_metadata_r207.tsv --load_version 207 --kbase_collection gtdb
-     gtdb_genome_attribs_loader.py bac120_metadata_r207.tsv ar53_metadata_r207.tsv --load_version 207 --kbase_collection gtdb --output  gtdb_genome_attribs.json
-"""
+optional arguments:
+  --kbase_collection KBASE_COLLECTION
+                        kbase collection identifier name (default: GTDB)
```

Comment:
Do you want to use the variable for GTDB rather than hardcoding?

Reply:
👍

Reply:
:+1: